controlplane/kubeadm/internal/etcd/util/set.go

```diff
@@ -0,0 +1,211 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Modified copy of k8s.io/apimachinery/pkg/util/sets/int64.go
+
+package util
+
+import (
+	"reflect"
+	"sort"
+)
+
+type empty struct{}
+
+// sets.UInt64 is a set of int64s, implemented via map[uint64]struct{} for minimal memory consumption.
+type UInt64 map[uint64]empty
```

Comment:
This package is called `util`, the example says `sets.UInt64`. Can we either rename the package, or rename the type to say it's a Set?

Reply:
```suggestion
type UInt64Set map[uint64]empty
```

Reply:
sure, fixed

Reply:
Thanks for fixing. The code I modified _came_ form a package called `sets`.